apcd-cms/src/apps/utils/apcd_database.py

```diff
@@ -958,7 +958,8 @@ def get_all_submissions_and_logs():
         """
         cur = conn.cursor()
         cur.execute(query)
-        return cur.fetchall()
+        results = [row[0] for row in cur.fetchall()]
```

Comment:
I think it would be more performant to make this a `cursor.fetchone()` call, or even `fetchmany`, since we're only using the first row.